lightning-tag/faq.md

```diff
@@ -140,4 +140,10 @@ Our servers will always return the full set of audiences that your user qualifie
 
 ## How can I prioritize which audiences are returned first?
 
-Provide a list of audiences you would like to return first to your Lotame representative, and they will prioritize them accordingly in the platform.
\ No newline at end of file
+Sequence your audiences being passed through an ad call to an ad server, such as Google Ad Manager. This ensures audiences will return in order of ranked priority, to avoid exclusion based on the character limitations of the ad server.
```

Comment:
Suggestion:
This ensures audiences will return in order of ranked priority avoiding exclusion based on the character limitations of the ad server.

Reply:
Agree on this tweak.